packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -121,6 +125,17 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
           onChange={setChecked}
         />
 
+        {allowance && !allowance.isZero() ? (
```

Comment:
nit: `allowance && !allowance.isZero() &&` instead of a ternary

Reply:
Let's update the message a bit:

```
You are seeing this dialog because the current allowance you have set for this token (x) is less than the amount you are trying to bridge (y).
```

Let's try putting in the blue information banner, see how it looks?